operator/pkg/ingress/service.go

```diff
@@ -43,6 +44,8 @@ type serviceManager struct {
 	ingressQueue workqueue.RateLimitingInterface
 }
 
+var allowedAnnotationPrefixes = []string{"service.beta.kubernetes.io", "service.kubernetes.io", "cloud.google.com"}
```

Comment:
@sayboras should we make this as a daemon flag? :thinking:

Reply:
That is good point :100:.

I did take a look at below docs to understand that list of annotations used by cloud providers. While right now seems like we have only these annotations, it's not guarantee to stay the same in future.

https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class

Reply:
@NikhilSharmaWe hope the above is not much a huddle for you.

Reply:
@NikhilSharmaWe did you push the changes? :thinking:

Reply:
@aanm Yes